guava/src/com/google/common/hash/Hashing.java

```diff
@@ -623,6 +631,7 @@ private ConcatenatedHashFunction(HashFunction... functions) {
     }
 
     @Override
+    @SuppressWarnings("argument.type.incompatible")//
```

Comment:
This warning suppression is too broad and doesn't have a justification. We should remove it and figure out why it's necessary before we merge this.